modules/loader/src/main/java/com/opengamma/strata/loader/csv/SecurityCsvLoader.java

```diff
@@ -92,19 +131,12 @@ static SecurityPosition parsePositionLightweight(CsvRow row, PositionInfo info,
       }
     } else {
       // simple
-      return parseSimple(row, info, resolver);
+      return parseBase(row, info, resolver);
     }
   }
 
-  /**
-   * Parses from the CSV row.
-   * 
-   * @param row  the CSV row
-   * @param info  the trade info
-   * @param resolver  the resolver used to parse additional information
-   * @return the parsed position
-   */
-  static SecurityPosition parseSimple(CsvRow row, PositionInfo info, PositionCsvInfoResolver resolver) {
+  // parses the base SecurityPosition
+  static SecurityPosition parseBase(CsvRow row, PositionInfo info, PositionCsvInfoResolver resolver) {
```

Comment:
I see we've got javadocs for the other static methods, could you add one for this one please?

Reply:
I feel like the method naming could be more explicative but can't really find a better alternative for this one and ```parseSimple()```. Up to you if you think you can improve them.